networkit/workflows.py

```diff
@@ -14,16 +14,18 @@
 
 def extractLargestComponent(G):
```

Comment:
These functions have a specific use case and I think are out of NetworKit's scope. We should deprecate them.
`extractLargestConnectedComponent` can be removed since it was deprecated more than two releases cycles ago.